README.md

```diff
@@ -263,7 +263,28 @@ kubectl apply -f hack/compute/apibinding.yaml
 registeredclusters                             singapore.open-cluster-management.io/v1alpha1   true         RegisteredCluster
 ```
 
-4. Create a registeredcluster CR in the kcp workspace
+4. Get the identityhash for synctarget resource in kcp
+```bash
+kubectl get apibindings workload.kcp.dev -o jsonpath='{.status.boundResources[?(@.resource=="synctargets")].schema.identityHash}'
+```
+And add the permissionClaim for synctarget and enter the identityhash which you got by executing above command in the apiexport `compute-apis` as follows
+```bash
+ - group: workload.kcp.dev
+   resource: synctargets
+   identityHash: <identityhash>
+```
+
+5. Create APIBinding to the compute-apis APIExport in the location workspace 
```

Comment:
I think this is the first time we reference a location workspace. We should add some clarification around the different workspaces in play.